### ISSUE TITLE
Error using shortcuts

### ISSUE THREAD

**Issue by leandro-hermes** [OpherV/gitflow4idea#322]
* **I'm submitting a ...**
  - [x] bug report

* **What is the current behavior?**

When I start a flow using shortcuts, an exception is thrown

* **How to reproduce:**

1. Configure a keyboard shortcut for an action, e.g. Start feature
2. Open a project
3. Use the shortcut
4. Close the project by menu File -> Close project
5. Open it again
6. Use the shortcut
7. See the exception in the notifications area

* **What is the expected behavior?**

Exception not to be thrown and the action to be executed.

* **Environment:**
  
  - Gitflow4idea version: *0.7.5*
  - Gitflow version: *1.12.3 (AVH Edition)*
  
  - *IntelliJ info:*
  ~~~~
  PhpStorm 2020.3.2
  Build #PS-203.7148.74, built on January 27, 2021
  Licensed to **_[hidden]_**
  Subscription is active until October 1, 2021.
  Runtime version: 11.0.9.1+11-b1145.77 amd64
  VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
  Linux 5.8.0-43-generic
  GC: ParNew, ConcurrentMarkSweep
  Memory: 1892M
  Cores: 8
  Registry: run.processes.with.pty=TRUE, js.debugger.webconsole=false
  Non-Bundled Plugins: String Manipulation, com.github.lppedd.idea-conventional-commit, com.wakatime.intellij.plugin, Gitflow, com.jetbrains.codeWithMe, com.intellij.plugins.html.instantEditing, com.jetbrains.plugins.jade, de.espend.idea.laravel, ru.adelf.idea.dotenv
  Current Desktop: ubuntu:GNOME
  ~~~~

* **Other information** 
Stack trace:
~~~
com.intellij.util.IncorrectOperationException: Sorry but parent: /home/leandro/<path> has already been disposed (see the cause for stacktrace) so the child: gitflow.GitflowConfigUtil$$Lambda$2526/0x0000000101a19c40@1a397dcb will never be disposed
	at com.intellij.openapi.util.ObjectTree.register(ObjectTree.java:55)
	at com.intellij.openapi.util.Disposer.register(Disposer.java:73)
	at gitflow.GitflowConfigUtil.getInstance(GitflowConfigUtil.java:60)
	at gitflow.ui.GitflowStartFeatureDialog.getDefaultBranch(GitflowStartFeatureDialog.java:22)
	at gitflow.ui.AbstractBranchStartDialog.<init>(AbstractBranchStartDialog.java:46)
	at gitflow.ui.GitflowStartFeatureDialog.<init>(GitflowStartFeatureDialog.java:12)
	at gitflow.actions.StartFeatureAction.actionPerformed(StartFeatureAction.java:28)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:281)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$1.performAction(IdeKeyEventDispatcher.java:615)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.lambda$processAction$3(IdeKeyEventDispatcher.java:675)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:95)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:674)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:625)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processActionOrWaitSecondStroke(IdeKeyEventDispatcher.java:522)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.inInitState(IdeKeyEventDispatcher.java:476)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:225)
	at com.intellij.ide.IdeEventQueue.dispatchKeyEvent(IdeEventQueue.java:892)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:837)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:454)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:773)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$9(IdeEventQueue.java:453)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:822)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:507)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
~~~

**Comment by OpherV** [comment]
Oh, great find! Thanks for reporting
I'll address this in the next version

**Comment by OpherV** [comment]
Fixed in 0.7.6
